### ISSUE TITLE
I want you to return to the previous UI

### ISSUE THREAD

**Issue by Tooka4540** [igrigorik/videospeed#839]
On youtube, the button is displayed above the video and it is hard to see.

I can speak only a little English. (google translate)

**Comment by Tooka4540** [comment]
![中央](https://user-images.githubusercontent.com/89531682/130812290-8340d846-7670-4097-ad2a-009397fe4102.png)

**Comment by Yash089610** [comment]
Yes, please. It's not the issue of the new UI, but the new way of finding video frame location. Please use the old method as it didn't have this issue.

**Comment by toppits** [comment]
yes, please. It also looks ugly with the huge padding and 100% border radius. Is that everyone does it, not enough?

**Comment by Longoon12000** [comment]
I do not mind the round corners, however for some reason the placement now is not in the top left anymore but somewhere randomly on the video and the icons in the buttons are not centered.
![vsc](https://user-images.githubusercontent.com/51454971/130826748-a6b8590a-b66a-43ea-9c50-2521c61c9a62.png)
Pls fix <3

**Comment by paulirish** [comment]
Yeah I think this is a dupe of #786, that @russelldavis fixed in #854 

so this can be closed.  

(However that fix is only here in the local dev version of the extension. not yet published to CWS)
